core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -21,19 +21,35 @@
  */
 
 oppia.factory('SimpleEditorManagerService', [
-  'explorationInitStateNameService', 'OutcomeObjectFactory',
-  'QuestionListObjectFactory', 'QuestionObjectFactory',
-  'SimpleEditorShimService', 'StatesToQuestionsService',
+  'AnswerGroupObjectFactory', 'explorationInitStateNameService',
+  'OutcomeObjectFactory', 'QuestionObjectFactory', 'QuestionListObjectFactory',
+  'RuleObjectFactory', 'StatesToQuestionsService', 'SimpleEditorShimService',
   function(
-    explorationInitStateNameService, OutcomeObjectFactory,
-    QuestionListObjectFactory, QuestionObjectFactory,
-    SimpleEditorShimService, StatesToQuestionsService) {
+      AnswerGroupObjectFactory, explorationInitStateNameService,
+      OutcomeObjectFactory, QuestionObjectFactory, QuestionListObjectFactory,
+      RuleObjectFactory, StatesToQuestionsService, SimpleEditorShimService) {
     var data = {
       title: null,
       introductionHtml: null,
       questionList: null
     };
 
+    var DEFAULT_INTERACTION_PROPERTIES = {
+      MultipleChoiceInput: {
+        CUSTOMIZATION_ARGS:  {
+          choices: {
+            value: ['<p>Option 1</p>']
+          }
+        },
+        ANSWER_GROUP: {
+          type: 'Equals',
```

Comment:
This looks more like a rule than an answer group. It might be worth naming it differently -- e.g. FIRST_ANSWER_GROUP_RULE.